src/CMakeLists.txt

```diff
@@ -218,7 +218,7 @@ else()
 endif ()
 target_compile_definitions(flameshot PRIVATE APP_PREFIX="${CMAKE_INSTALL_PREFIX}")
 target_compile_definitions(flameshot PRIVATE APP_VERSION="v${PROJECT_VERSION}")
-target_compile_definitions(flameshot PRIVATE IMGUR_CLIENT_ID="313baf0c7b4d3ff")
+target_compile_definitions(flameshot PRIVATE IMGUR_CLIENT_ID="43467546c66193c")
```

Comment:
Is this necessary? Why?

Reply:
Hey @mmahmoudian 
It's not necessary at all, I apologise for this change, I was testing multiple image upload and I was facing too many requests error from Imgur so I changed client ID for my test cases, I will just remove this right away.